src/puppetlabs/trapperkeeper/services/metrics/metrics_core.clj

```diff
@@ -1,6 +1,9 @@
 (ns puppetlabs.trapperkeeper.services.metrics.metrics-core
-  (:import (com.codahale.metrics JmxReporter MetricRegistry))
+  (:import (com.codahale.metrics JmxReporter MetricRegistry)
+           com.fasterxml.jackson.core.JsonParseException)
```

Comment:
I don't think that's valid syntax for importing a class, unless I'm missing something. Wrapping it in parentheses gives you this compilation error,

```
Exception in thread "main" java.lang.IllegalArgumentException: Unable to resolve classname: JsonParseException, compiling:(puppetlabs/trapperkeeper/services/metrics/metrics_core.clj:99:19)
	at clojure.lang.Compiler.analyzeSeq(Compiler.java:6730)
	at clojure.lang.Compiler.analyze(Compiler.java:6524)
	at clojure.lang.Compiler.analyze(Compiler.java:6485)
	at clojure.lang.Compiler$BodyExpr$Parser.parse(Compiler.java:5861)
	at clojure.lang.Compiler$FnMethod.parse(Compiler.java:5296)
	at clojure.lang.Compiler$FnExpr.parse(Compiler.java:3925)
	at clojure.lang.Compiler.analyzeSeq(Compiler.java:6721)
	at clojure.lang.Compiler.analyze(Compiler.java:6524)
```